interface_specifications/pe_openapi/apis/v1/a1_pe_invitation_get.md

```diff
@@ -0,0 +1,28 @@
+### Invitation GET
+
+`GET /pe/v1/invitation?_oobid=5f0e3ffb-3f92-4648-9868-0d6f8889e6f3`
```

Comment:
Do we want to do this more in REST style? So invitation -> invitations and instead of using a query param, to have the id in the Path
/pe/v1/invitations/5f0e3ffb-3f92-4648-9868-0d6f8889e6f3

Reply:
This example was followed in this call:

Page:   
 `https://github.com/hyperledger/aries-rfcs/tree/master/features/0434-outofband`

Example:

`http://example.com/ssi?oob=eyJAdHlwZSI6Imh0dHBzOi8vZGlkY29tbS5vcmcvb3V0LW9mLWJhbmQvMS4wL2ludml0YXRpb24iLCJAaWQiOiI2OTIxMmEzYS1kMDY4LTRmOWQtYTJkZC00NzQxYmNhODlhZjMiLCJsYWJlbCI6IkZhYmVyIENvbGxlZ2UiLCAiZ29hbF9jb2RlIjoiaXNzdWUtdmMiLCJnb2FsIjoiVG8gaXNzdWUgYSBGYWJlciBDb2xsZWdlIEdyYWR1YXRlIGNyZWRlbnRpYWwiLCJoYW5kc2hha2VfcHJvdG9jb2xzIjpbImh0dHBzOi8vZGlkY29tbS5vcmcvZGlkZXhjaGFuZ2UvMS4wIiwiaHR0cHM6Ly9kaWRjb21tLm9yZy9jb25uZWN0aW9ucy8xLjAiXSwic2VydmljZSI6WyJkaWQ6c292OkxqZ3BTVDJyanNveFllZ1FEUm03RUwiXX0`

Reply:
fixed